packages/manager/modules/pci/src/projects/project/data-processing/notebooks/data-processing-notebooks.html

```diff
@@ -0,0 +1,145 @@
+<div data-ui-view>
+    <div>
+        <!-- cui messages -->
+        <cui-message-container
+            data-messages="$ctrl.messages"
+        ></cui-message-container>
+        <oui-datagrid
+            data-rows="$ctrl.notebooks"
+            data-refreshable
```

Comment:
**nitpick:** `oui-datagrid` does not expect `refreshable` attribute. Could you please remove it?
ref: https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/datagrid/src/js/datagrid.directive.js

Reply:
Fixed :)